backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/ToplevelPhases.kt

```diff
@@ -450,6 +485,8 @@ private val backendCodegen = namedUnitPhase(
         name = "Backend codegen",
         description = "Backend code generation",
         lower = takeFromContext<Context, Unit, IrModuleFragment> { it.irModule!! } then
+                preInlinePhase then
+                dependenciesPreInlinePhase then
```

Comment:
Does this approach work with caches? When `inline` function comes from cached library.

Reply:
I'll check.